src/cmd/report.go

```diff
@@ -19,42 +19,43 @@ var (
 	dhFlag   string
 	oFlag    string
 
-	// fetchCmd represents the fetch command
-	fetchCmd = &cobra.Command{
-		Use:   "fetch",
-		Short: "Fetch install data from Brew, GitHub, and/or Docker Hub",
+	// reportCmd represents the report command
+	reportCmd = &cobra.Command{
+		Use:     "report",
+		Aliases: []string{"fetch"},
+		Short:   "report install data from Brew, GitHub, and/or Docker Hub",
```

Comment:
Fetch and report aren't drop in replacements. I think we should reword this now as it sounds weird as is.